lib/muontrap/options.ex

```diff
@@ -125,6 +126,9 @@ defmodule MuonTrap.Options do
   defp validate_option(:daemon, {:logger_metadata, metadata}, opts) when is_list(metadata),
     do: Map.put(opts, :logger_metadata, metadata)
 
+  defp validate_option(:daemon, {:custom_logger, logger}, opts) when is_function(logger, 1),
```

Comment:
It just occurred to me that this should probably also accept an MFA tuple, so I'll make that update.